test/e2e/v1beta1/scripts/setup-katib.sh

```diff
@@ -76,6 +76,13 @@ if [ $? -ne 1 ]; then
   exit 1
 fi
 
+# TODO (tenzen-y): Once the changes on https://github.com/kubeflow/testing/pull/974 are reflected in the `public.ecr.aws` registry, we must remove this process.
+# To avoid the `../../../../pkg/mod/k8s.io/client-go@v0.22.2/plugin/pkg/client/auth/exec/metrics.go:21:2: package io/fs is not in GOROOT (/usr/local/go/src/io/fs)` error,
```

Comment:
Also, wondering why this error is happening only for Golang version <1.6?

Reply:
The `io/fs` package was introduced in Go 1.16.

> File Systems
The new [io/fs](https://go.dev/pkg/io/fs/) package defines the [fs.FS](https://go.dev/pkg/io/fs/#FS) interface, an abstraction for read-only trees of files. The standard library packages have been adapted to make use of the interface as appropriate.

https://go.dev/doc/go1.16#library